app/models/topic_tracking_state.rb

```diff
@@ -464,7 +464,6 @@ def self.report_raw_sql(
       JOIN user_options AS uo ON uo.user_id = u.id
       JOIN categories c ON c.id = topics.category_id
```

Comment:
Sorry while you're knees deep in the topic tracking state code, do you happen to know why we need to join against `categories` here? It doesn't seem to be used at all.